### PR TITLE
Clean up files

### PR DESCRIPTION
**Pull request by yudetamago**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## How to get started
+
+Python 3.6 is required.
+For settting up Python 3 environment, [virtualenv](https://virtualenv.pypa.io/en/stable/) is commonly used, like:
```
Comment (nrryuya) [inline tests/README.md]: I personal opinion about python environment manager is that basically standard `venv` is the first option and if you want to use various python version then `pyenv` + `pyenv+virtualenv` is good.

Comment (nrryuya) [reply tests/README.md]: Anyway, I don't want to explain the python environment management here so how about just recommending readers to make their own environment somehow.

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+. venv/bin/activate
+```
+
+Then install required packages:
+
+```
+pip install -r requirements.txt
+```
+
+## Running the tests
```
Comment (nrryuya) [inline tests/README.md]: Instruction to select one test or just one functions is useful.
```
pytest test_erc721.py
```
```
pytest test_erc721.py::test_approve
```

File: README.md [hunk 3]
```diff
@@ -6,72 +6,12 @@ Formally Verified Vyper Contracts
 FVyper is a collection of formally verified, useful Vyper programs.  
 See [roadmap](https://github.com/LayerXcom/verified-vyper-contracts/issues/5).
 
-## How to get started
+## Structure of this project
```
Comment (nrryuya) [inline README.md]: `Directory structure` ?

File: README.md [hunk 4]
```diff
@@ -6,72 +6,12 @@ Formally Verified Vyper Contracts
...
+The `/contracts` directory contains vyper contracts we use in formal verification (`/k` directory) and unit testing ( `/tests` directory).
 
-You need to install dependencies of K. See [prerequisites](https://github.com/kframework/k#prerequisites).
+The `/k` directory contains formal verification specs using [K Framework](https://github.com/kframework/k).
```
Comment (nrryuya) [inline README.md]: ```The `/k` directory contains files to do formal verification with [K Framework]```

File: tests/README.md (hunk 1 above)
Comment (yudetamago) [reply tests/README.md]: Yes, it is just recommendation, so removing it is OK.
Some other projects like [py-evm](
https://github.com/ethereum/py-evm#development) explain about venv, then I added it.

(I also use `pyenv` + `pyenv+virtualenv` :) )

File: tests/README.md (hunk 2 above)
Comment (yudetamago) [reply tests/README.md]: yup!

File: README.md (hunk 3 above)
Comment (yudetamago) [reply README.md]: `Directory structure` is easy to understand, OK!

**Pull request merged**